sdk/analysisservices/Azure.ResourceManager.AnalysisServices/api/Azure.ResourceManager.AnalysisServices.netstandard2.0.cs

```diff
@@ -122,15 +166,21 @@ public partial class CheckServerNameAvailabilityContent
     public partial class CheckServerNameAvailabilityResult
     {
         internal CheckServerNameAvailabilityResult() { }
+        public bool? IsNameAvailable { get { throw null; } }
         public string Message { get { throw null; } }
-        public bool? NameAvailable { get { throw null; } }
         public string Reason { get { throw null; } }
     }
     public enum ConnectionMode
     {
         All = 0,
         ReadOnly = 1,
     }
+    public partial class ExistingResourceSkuDetails
+    {
+        internal ExistingResourceSkuDetails() { }
+        public string ResourceType { get { throw null; } }
```

Comment:
Can this property use `ResourceType` type?

Reply:
fixed